db/migrate/20160914191645_add_status_to_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToRequests < ActiveRecord::Migration
+  def change
+    add_column :requests, :status, :integer
```

Comment:
Anyway if you change the mapping, if you already have data in the database, you are going to need to update those if you want to update the default value..